recipes-support/collision-avoidance-library/libcoav_0.1.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Intel Colision Avoidance Library"
+AUTHOR = "Intel Corporation"
+SECTION = "libs"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
+
+DEPENDS = "glm python-future librealsense"
+RDEPENDS_${PN} = "librealsense"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "gitsm://github.com/01org/collision-avoidance-library.git;protocol=https;branch=master"
+
+S = "${WORKDIR}/git"
+
+inherit cmake pythonnative
+
+EXTRA_OECMAKE = "-DWITH_REALSENSE=ON -DWITH_GAZEBO=OFF -DWITH_TOOLS=OFF -DWITH_VDEBUG=OFF"
+
+PACKAGES = "${PN}-dbg ${PN}-dev ${PN}"
+
+FILES_${PN}-dbg += "${bindir}/.debug"
+
+FILES_${PN}-dev += "\
+    ${libdir}/*.so \
+    ${includedir}/${PN} \
+"
+
+FILES_${PN} += "\
+    ${libdir}/*.so.0.* \
+"
+
+# avoid packaging error
+INSANE_SKIP_${PN} = "ldflags"
```

Comment:
I don't have time to dig into this right now but seems odd as default cmake build flags should not force you to work around ldflags errors.